
Some crap, like gpm.h does use this header.

  - Rene Rebe <rene@exactcode.de>

--- /dev/null	1970-01-01 01:00:00.000000000 +0100
+++ dietlibc-0.28/include/sys/fcntl.h	2005-02-07 12:34:33.300839392 +0100
@@ -0,0 +1,4 @@
+
+#warning "This is a non-standard header - use <fcntl.h> instead!"
+#include <fcntl.h>
+
